### DISCUSSION TITLE
Is it possible to enable converting documents into PDF for offline reading

### DISCUSSION THREAD

**Discussion by mrRobot62** [textX/textX#396] [Q&A]
That would be very helpful maybe not only for me :-).
Not one page after the other. Exporting from root and print everything

**Comment by igordejanovic** [comment]
textX uses [mkdocs](https://www.mkdocs.org/) for its documentation. You can read the docs offline by installing mkdocs and running 
```
mkdocs serve
```
in the root of the textX repo.


I haven't tried it myself but there are couple of mkdocs pdf exporters (e.g. [this](https://github.com/zhaoterryy/mkdocs-pdf-export-plugin) or [this](https://github.com/orzih/mkdocs-with-pdf)). I guess that could do it if you want pdf.  If you try it please let us know is it working.

**Reply by mrRobot62** [reply]
thanks for your reply - will check this next days :-)
